protocols/v2/roles-logic-sv2/src/job_creator.rs

```diff
@@ -275,6 +275,7 @@ fn coinbase(
 
```

Comment:
All of the changes below fix various unused warnings that were spamming the output when running tests.